src/service/layers-impl.js

```diff
@@ -577,9 +665,20 @@ export class LayoutElement {
       return;
     }
 
+    const element = this.element_;
+    const win = /** @type {!Window } */ (dev().assert(
+        element.ownerDocument.defaultView));
+    // If it remains fixed, it will still be a layer.
+    if (computedStyle(win, element).position === 'fixed') {
```

Comment:
How can we ensure this is called in a measure cycle (without interleaved mutates)?